cleanrl/pqn.py

```diff
@@ -0,0 +1,274 @@
+# docs and experiment results can be found at https://docs.cleanrl.dev/rl-algorithms/dqn/#dqnpy
+import os
+import random
+import time
+from collections import deque
+from dataclasses import dataclass
+
+import envpool
```

Comment:
What do you think on making this more similar to `ppo.py` using gymnasium? Envpool doesn't work on windows for example (I think)

<img width="1025" alt="image" src="https://github.com/user-attachments/assets/f213ff54-803b-45fd-8754-7cb727cef0d3">

Reply:
Done! 👍